sonarts-core/src/rules/noIdenticalFunctionsRule.ts

```diff
@@ -67,8 +67,7 @@ export class Rule extends tslint.Rules.AbstractRule {
   }
 
   private static message(functionBlock: ts.Block): string {
-    const lineOfOriginalFunction =
-      lineAndCharacter(functionBlock.parent!.getStart(), functionBlock.getSourceFile()).line + 1;
+    const lineOfOriginalFunction = startLineAndCharacter(functionBlock.parent!).line + 1;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary not-null assertion. ([typescript:S4325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-489&issues=AWGz_GixUhrvXJpox_lb&open=AWGz_GixUhrvXJpox_lb)

[ISSUE_KEY]: AWGz_GixUhrvXJpox_lb